merkletree/pad_test.go

```diff
@@ -264,10 +264,23 @@ func TestTB(t *testing.T) {
 	// create next epoch and see if the TB is inserted as promised:
 	pad.Update(nil)
 
-	ap, err := pad.Lookup(key1)
+	ap, err := pad.Lookup(key)
 	if !bytes.Equal(ap.LookupIndex, tb.Index) || !bytes.Equal(ap.Leaf.Value(), tb.Value) {
 		t.Error("Value wasn't inserted as promised")
 	}
+	// step 1. verify VRF index
+	if !bytes.Equal(vrfPrivKey1.Compute([]byte(key)), ap.LookupIndex) {
```

Comment:
Done in https://github.com/coniks-sys/coniks-go/pull/54/commits/95650ab818b2b1e74de76175c305b8522e5aaf74